src/main/java/com/bloxbean/cardano/client/util/Tuple.java

```diff
@@ -1,11 +1,28 @@
 package com.bloxbean.cardano.client.util;
 
+import java.util.Objects;
```

Comment:
@satran004 what's the best way of fixing this? 

I need `equals` and `hashcode` for the grouping to work properly. Should we introdue lombok instead?

Or leave it as is and create a test with the description so that if they get deleted the test will remind why we need them?

Reply:
@nemo83 We can introduce Lombak if that solves your problem. We are using Lombok in other places.